blocks/bricks/attention.py

```diff
@@ -325,9 +325,9 @@ def __init__(self, match_dim, state_transformer=None,
         self.energy_computer = energy_computer
 
         children = [self.state_transformers, attended_transformer,
-                    energy_computer] + kwargs.get('children', [])
-        super(SequenceContentAttention, self).__init__(children=children,
-                                                       **kwargs)
+                    energy_computer]
+        kwargs.setdefault('children', []).extend(children)
```

Comment:
Yep, this is a variation on a pretty common idiom with `setdefault`: http://python.net/~goodger/projects/pycon/2007/idiomatic/handout.html#dictionary-setdefault-method-1